repairnator/repairnator-pipeline/src/test/resources/prophet4j/_BIN/README.md

```diff
@@ -0,0 +1,14 @@
+Prophet4J is inside the Coming lib, and is introduced to compute OverfittingScores for patches.
```

Comment:
seems the same content as above

Reply:
Yes.

Reply:
should be deduplicated then

Reply:
For this version of Coming, code files in `src/main/...` and `src/test/...` would access their respective resources files. Therefore, the duplications should be necessary, and also these files are rather tiny.